lib/utils/getImportantPosition.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+/**
+ * @typedef {{ index: number, endIndex: number }} Position
+ * @param {string} source
+ * @returns {Position | undefined}
+ */
+module.exports = function getImportantPosition(source) {
+	const pattern = /!\s*important\b/gi;
+	const match = pattern.exec(source);
+
+	if (!match) return;
+
+	return { index: match.index, endIndex: pattern.lastIndex };
+};
```

Comment:
[note] I cannot find how to get a position of `!important` from CSS source code via `postcss` or `postcss-value-parser`. So, I've written the tiny utility `getImportantPosition`.

If anyone knows a better way, please let me know.